src/test/java/com/wikia/webdriver/testcases/adstests/TestUap.java

```diff
@@ -17,6 +17,10 @@
 import java.util.Map;
 
 public class TestUap extends TemplateNoFirstLoad {
+public final static String MOBILE_IN_CONTENT_CONTAINER = ".mobile-in-content";
+public final static String MOBILE_PREFOOTER_CONTAINER = ".mobile-prefooter";
+public final static String MOBILE_BOTTOM_LEADERBOARD_CONTAINER = ".mobile-bottom-leaderboard";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)